### DISCUSSION TITLE
КАК СКРЫТЬ статистику ТОНА

### DISCUSSION THREAD

**Discussion by OlegSivkov** [Lolliedieb/lolMiner-releases#1433] [Q&A]
Добрый день , меня интересует как убрать статистику Тона при майнинге , пусть в статистике Hive будет указана добыча эфира , но добычу Тона желаю скрыть, и так же убрать статистику Sha -256 какую команду куда прописать ? пусть 
это будет скрытый майнинг ,  я на пуле узнаю что и сколько намайнило , но не желаю видеть в Hive

**Comment by GizDS** [comment]
--statsformat GpuName,Algo,Speed,PoolHr,Shares,SharesPerMin,CoreT,JuncT,MemT,FanPct,Power,CoreClk,MemClk,LhrLks,LhrParam

другие параметры можно найти здесь [https://github.com/Lolliedieb/lolMiner-releases/wiki/Stats-Format-(English)](url)

Если нужна статистика по dual-монете, то:

--statsformat GpuName,Algo,Speed,PoolHr,Shares,SharesPerMin,Speed2,PoolHr2,Shares2,CoreT,JuncT,MemT,FanPct,Power,CoreClk,MemClk,LhrLks,LhrParam

(в некоторых старых версиях параметр LhrLks не дает запустить майнер)
